build-system/common/git.js

```diff
@@ -20,39 +20,19 @@
  */
 
 const {
-  isTravisBuild,
   isTravisPullRequestBuild,
   travisPullRequestBranch,
   travisPullRequestSha,
 } = require('./travis');
 const {getStdout} = require('./exec');
 
 /**
- * Returns the commit at which the current branch was forked off of master.
- * On Travis, there is an additional merge commit, so we must pick the first of
- * the boundary commits (prefixed with a -) returned by git rev-list.
- * On local branches, this is merge base of the current branch off of master.
+ * Returns the remote tracking branch associated with the local master branch.
  * @return {string}
  */
-function gitBranchCreationPoint() {
-  if (isTravisBuild()) {
-    const traviPrSha = travisPullRequestSha();
-    return getStdout(
-      `git rev-list --boundary ${traviPrSha}...master | grep "^-" | head -n 1 | cut -c2-`
-    ).trim();
-  }
-  return gitMergeBaseLocalMaster();
-}
-
-/**
- * Returns the `master` parent of the merge commit (current HEAD) on Travis.
- * Note: This is not the same as origin/master (a moving target), since new
- * commits can be merged while a Travis build is in progress.
- * See https://travis-ci.community/t/origin-master-moving-forward-between-build-stages/4189/6
- * @return {string}
- */
-function gitTravisMasterBaseline() {
```

Comment:
This is used by the bundle-size and visual-diff checks. I'm going to let @danielrozenberg give this a careful review to see if the replacement is equivalent, and I too will take a proper look tomorrow.

Reply:
This change is no longer included in the updated PR. If there are still code paths that need extra scrutiny, could you comment on the updated code? Marking resolved.